build.gradle

```diff
@@ -86,110 +87,80 @@ test {
-          }
-          licenses {
-            license {
-              name = 'The Apache Software License, Version 2.0'
-              url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
-              distribution = 'repo'
-            }
+        issueManagement {
+          system = 'GitHub'
+          url = 'https://github.com/sonatype-nexus-community/scan-gradle-plugin/issues'
+        }
+        licenses {
+          license {
+            name = 'The Apache Software License, Version 2.0'
+            url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+            distribution = 'repo'
           }
-          scm {
-            connection = 'scm:git:git://github.com/sonatype-nexus-community/scan-gradle-plugin.git'
-            developerConnection = 'scm:git:ssh://git@github.com:sonatype-nexus-community/scan-gradle-plugin.git'
-            url = 'https://github.com/sonatype-nexus-community/scan-gradle-plugin'
+        }
+        scm {
+          connection = 'scm:git:git://github.com/sonatype-nexus-community/scan-gradle-plugin.git'
+          developerConnection = 'scm:git:ssh://git@github.com:sonatype-nexus-community/scan-gradle-plugin.git'
+          url = 'https://github.com/sonatype-nexus-community/scan-gradle-plugin'
+        }
+        developers {
+          developer {
+            name = 'Guillermo Varela'
           }
-          developers {
-            developer {
-              name = 'Guillermo Varela'
-            }
-            developer {
-              name = 'Usman Shaikh'
-            }
+          developer {
+            name = 'Usman Shaikh'
           }
         }
-
-        artifactId = project.name
       }
     }
-
-    // configure signing if performing release
-    if ('do-release' == System.getenv('CIRCLE_JOB')) {
-      signing {
-        def final encodedKey = System.getenv('SECRING_GPG_ASC_BASE64')
-        def final signingKey = new String(encodedKey.decodeBase64())
-
-        allprojects {
-          println 'setup gpg signing with in-memory key'
-          useInMemoryPgpKeys(
-              signingKey,
-              System.getenv('GPG_PASSPHRASE')
-          )
+  }
+  repositories {
+    maven {
+      url = layout.buildDirectory.dir('staging-deploy')
+    }
+  }
+}
+jreleaser {
+  signing {
+    active = 'ALWAYS'
+    armored = true
+  }
+  deploy {
+    maven {
+      mavenCentral {
+        sonatype {
+          active = 'ALWAYS'
+          url = 'https://central.sonatype.com/api/v1/publisher'
+          stagingRepository('build/staging-deploy')
         }
-
-        sign publishing.publications.pluginMaven
-        sign publishing.publications.scanPluginMarkerMaven
       }
     }
   }
 }
 
+// Configure signing
+signing {
+  useInMemoryPgpKeys(System.getenv('GPG_SIGNING_KEY'), System.getenv('GPG_PASSPHRASE'))
+  sign publishing.publications.maven
+
+}
+
+
 afterReleaseBuild.dependsOn publish
 publish.finalizedBy closeAndReleaseRepository
```

Comment:
This is possibly no longer needed as I believe is from the old publishing plugin

Reply:
[Commit](https://github.com/sonatype-nexus-community/scan-gradle-plugin/pull/174/commits/905e23895bc5553b1a239aa45cd852585ad6d599)